src/main/java/br/com/sbk/sbking/core/HandEvaluations.java

```diff
@@ -79,4 +79,9 @@ public boolean isBalanced() {
         && this.getNumberOfDoubletonSuits() <= 1;
   }
 
+  public boolean hasEightOrMoreCardsInAnySuit() {
+    int longestSuit = this.getCardsPerSuit().entrySet().stream().map(Entry::getValue).reduce(0, Math::max);
```

Comment:
You can use `valueSet`directly instead of getting all entries and them mapping to the value.

Reply:
Also, you forgot to test this.

Reply:
A couldn't find a `valueSet` method for the map, instead I used `.values()`.

Reply:
I know this is one of the methods that are beeing repeated, I'll solve it in the 2-weak PR